elastic/kibana
# [Fleet] hiding log callout for managed policy

## Summary

Fix https://github.com/elastic/kibana/issues/118498

Hiding disabled logs callout for managed policies, because in cloud, managed fleet server policy log changes are not allowed to be changed.

To test:

1. On prem:
- Check any policy on-prem e.g. `Default Fleet Server policy`
- Disable logs and metrics collection
- Check on Logs tab that callout is still visible
![image](https://user-images.githubusercontent.com/90178898/142649177-5b6f9adb-9d7f-4cd5-a4fa-d06f45c52479.png)


2. Cloud
- Check `Elastic Cloud agent policy` on cloud 
- Disable logs and metrics
- Check that callout is not visible
TODO: find a way to test this without merging
